Tools/CISupport/ews-build/steps.py

```diff
@@ -2168,7 +2167,7 @@ def send_email_for_new_build_failure(self):
             if pr_number and not self.should_send_email_for_pr(pr_number):
                 return
             if not patch_id and not (pr_number and sha):
-                print('Unrecognized change type')
+                self._addToLog('Unrecognized change type')
```

Comment:
I don't think we will ever hit it. Some other steps will check similar check fail much earlier. But that's ok print to UI here as well.